phpBB/phpbb/auth/provider/oauth/oauth.php

```diff
@@ -828,8 +827,8 @@ protected function is_set_code($service)
 	 * Sets a redirect to the authorization uri.
 	 *
 	 * @param OAuth1Service|OAuth2Service $service		The external OAuth service
-	 * @return array|false								Array if an error occurred,
-	 *                            						false on success
+	 * @return array									Array if an error occurred,
```

Comment:
array|never
that type was introduced in 8.1, I don't know if there is any problem if we put it in the annotations